src/avkit.cs

```diff
@@ -546,6 +570,34 @@ interface AVPlayerView {
 		[Mac (12,0)]
 		[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
 		NSObject WeakDelegate { get; set; }
+
+		[Mac (13,0), NoWatch, NoiOS, NoMacCatalyst, NoTV]
+		[Export ("speeds", ArgumentSemantic.Copy)]
+		AVPlaybackSpeed[] Speeds { get; set; }
+
+		[Mac (13,0), NoWatch, NoiOS, NoMacCatalyst, NoTV]
+		[NullAllowed, Export ("selectedSpeed")]
+		AVPlaybackSpeed SelectedSpeed { get; }
+
+		[Mac (13,0), NoWatch, NoiOS, NoMacCatalyst, NoTV]
+		[Export ("selectSpeed:")]
+		void SelectSpeed (AVPlaybackSpeed speed);
+
+		[NoWatch, NoTV, NoMacCatalyst, NoiOS, Mac (13, 0)]
+		[Export ("allowsVideoFrameAnalysis")]
+		bool AllowsVideoFrameAnalysis { get; set; }
+
+		[Mac (13,0), NoWatch, NoiOS, NoMacCatalyst, NoTV]
+		[Export ("allowsMagnification")]
+		bool AllowsMagnification { get; [Bind ("setAllowsMagnification:")] set; }
+
+		[Mac (13,0), NoWatch, NoiOS, NoMacCatalyst, NoTV]
+		[Export ("magnification")]
+		nfloat Magnification { get; [Bind ("setMagnification:")] set; }
+
+		[Mac (13,0), NoWatch, NoiOS, NoMacCatalyst, NoTV]
+		[Export ("setMagnification:centeredAtPoint:")]
+		void SetMagnification (nfloat magnification, CGPoint point);
```

Comment:
Providing more info to the developer is always preferrable:

```suggestion
		void SetMagnification (nfloat magnification, CGPoint centeredAtPoint);
```